### PR TITLE
Update frontmatter param to hide_menu

### PR DESCRIPTION
**Pull request by qkombur**
Old value is obsolete after a new docs page was released.

This new value will make Grafana release notes appear in the "What's new / Release notes" section on https://grafana.com/docs/

### CODE REVIEW

File: update-changelog/writeDocsFiles.ts [hunk 1]
```diff
@@ -14,8 +14,7 @@ export async function writeDocsFiles({ version, builder }: DocsReleaseNotesWrite
 
 	const content = `+++
 title = "${title}"
-[_build]
-list = false
+hide_menu = false
```
Comment (torkelo) [inline update-changelog/writeDocsFiles.ts:17]: Should this not be true?

Comment (qkombur) [reply update-changelog/writeDocsFiles.ts:17]: If my understanding is correct the Grafana docs no longer use  `[_build]
list = true` in the hugo templates at all. We just need hide_menu set to false now.

For reference: https://github.com/grafana/website/pull/4809 and https://github.com/grafana/website/pull/7390

Comment (robbymilo) [reply update-changelog/writeDocsFiles.ts:17]: `hide_menu` should be set to true, so the page doesn't show in the left menu.

We need `_build.list` set to true (or removed) in order to query the release notes for the docs homepage.

Comment (qkombur) [reply update-changelog/writeDocsFiles.ts]: Ahhh I understand now. Thank you for clarifying.

**Review by robbymilo — APPROVED** [review]
Looks good to me but would like someone from the grafana team to take a look as well.

**Review by marefr — APPROVED** [review]
Just pushed a commit, needs to compile typescript to javascript as well. LGTM

**Review by tolzhabayev — APPROVED** [review]
What @marefr said :)

**Pull request merged**